sops/amq-scaling.md

```diff
@@ -0,0 +1,46 @@
+# AMQ Online Scaling Guide
+The purpose of this guide is to outline the existing scaling capabilities of AMQ Online, including scaling options for high availability and performance driven implementations along with some known limitations within the product.
+
+
+## AMQ Online Components 
+The basic AMQ Online infrastructure consists of the following components: 
+ - Operator
+ - Address-Space-Controller 
+ - Api-server,
+ - Console
+ - (optional) auth-service.
+
+ There is no scaling options for the above components. It is rare that these components will be under load. The following components make up the messaging infrastructure which will be exposed to load under use:
```

Comment:
`There are no scaling options` ?
I am not sure I would add `It is rare that these components will be under load` here (but only my 2 cents). One could ask then what happens when they are.

Reply:
Yeah agree, just was unsure best how to word it. Ill remove it :+1: